packages/plugin-core/src/test/suite-integ/CompletionProvider.test.ts

```diff
@@ -43,7 +43,7 @@ suite("completionProvider", function () {
       test("THEN provide completions", async () => {
         const { engine, vaults } = ExtensionProvider.getDWorkspace();
         // Open a note, add [[]]
-        await WSUtils.openNote(
+        await new WSUtilsV2(ExtensionProvider.getExtension()).openNote(
```

Comment:
nitpick: Shouldn't this be: 

```suggestion
        await ExtensionProvider.getExtension().wsUtils().openNote(
```

instead of creating a new `WSUtilsV2` instance?

Reply:
Same for all remainder of test changes.

Reply:
@hikchoi - I think we want to bring this forward into the upcoming release, so @kevinslin - want to take this one up as a later follow up?  I'm going to go ahead and merge / push to v100